ddpui/api/chat_with_data_api.py

```
@@ -0,0 +1,93 @@
+    scope_type="dashboard" + scope_id = chat restricted to that dashboard."""
+    orguser: OrgUser = request.orguser
+    try:
+        session = service.create_session(orguser, payload)
+    except InvalidScope as err:
+        raise HttpError(400, str(err)) from err
+    return api_response(success=True, data=SessionOut.from_model(session))
+
+
+@chat_with_data_router.get("/sessions/")
+@has_permission(["can_use_chat_with_data"])
+def list_sessions(request, scope_type: str = None):
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)